src/sbt-test/sbt-databricks/classpath/build.sbt

```diff
@@ -14,8 +14,6 @@ lazy val dbcSettings: Seq[Setting[_]] = Seq(
   dbcPassword := "test"
 )
 
-dbcSettings
```

Comment:
Required to make sure projC does not include these settings :) Not sure why it was here before.